src/index.js

```diff
@@ -0,0 +1,142 @@
+let currentTemperature = 0;
+
+const tempValue = document.getElementById('tempValue');
+
+const temperatureDisplay = () => {
+    tempValue.textContent = currentTemperature;
+    tempColor();
+};
+
+const addDegree = () => {
+    currentTemperature += 1;
+    tempValue.textContent = currentTemperature;
+    temperatureDisplay();
+};
+
+const subtractDegree = () => {
+    currentTemperature -= 1;
+    tempValue.textContent = currentTemperature;
+    temperatureDisplay();
+};
+
+const tempColor = () => {
```

Comment:
Would recommend a more descriptive name that captures what this function does (which it actually also deals with changing the landscape too, not just color).

```suggestion
const updateTempDisplay = () => {
// or updateTempColorAndLandscape
```